README.md

```
@@ -493,12 +457,12 @@ Connect AutoMem to cloud AI platforms via HTTPS. Works with:
 - **Claude mobile app**
 - **ElevenLabs Agents**
 
-See [MCP over SSE documentation](docs/MCP_SSE.md) for setup instructions.
+See [Remote MCP documentation](docs/MCP_SSE.md) for setup instructions.
 
 👉 **Resources**:
 
 - NPM bridge (local): https://www.npmjs.com/package/@verygoodplugins/mcp-automem
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)